assets/js/app.js

```diff
@@ -0,0 +1,97 @@
+const myLibrary = [];
+
+function Book(title, author, pages, status) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.status = status;
+}
+
+function addBookToLibrary(book) {
+  myLibrary.push(book);
+}
+
+function deleteBookFromLibrary(index) {
+  myLibrary.splice(index, 1);
+}
+
+function updateStatusInLibrary(index, status) {
```

Comment:
`OPTIONAL` Since this is a toggle function for the status of the book another way to implement it would've been
```
 myLibrary[index].status =  myLibrary[index].status!;
```